README.md

```diff
@@ -6,31 +6,53 @@
-execution state, both for the current live sporks and for past sporks. It also serves as a basis for the implementation
-of the Rosetta Data API, used in the larger blockchain ecosystem as a common generic interface for blockchain integration.
+execution state, both for the current live sporks and for past sporks.
 
 The state of past sporks is indexed by reading an execution node's protocol state and state trie write-ahead log.
 Optionally, a root checkpoint can be used to bootstrap state before a spork's start. In more specific terms, indexing
 of past sporks requires a Badger key-value database containing the Flow protocol state of the spork and a LedgerWAL with
 all the trie updates that happened on the spork.
 
-Indexing the live spork works similarly. The DPS will connect to the publish socket of an execution node that has
-whitelisted it and subscribe to state trie and transaction event updates. At the same time, the DPS will use access nodes
-to assemble a view of the finalized blockchain state. By combining these two sources of information, it can reconstruct
-the execution state on-the-fly.
+Indexing the live spork works similarly, but it reads the protocol state by acting as a consensus follower and reading
+block records from a Google Cloud Storage bucket.
 
-The Flow DPS maintains multiple specialized indexes for different purposes. One index is used for accessing the entire
-execution state at any given height, while another is used to follow the history of a specific Ledger register over time.
+The Flow DPS maintains multiple specialized indexes for different purposes.
 Contrary to the execution node's state trie, the indexes allow random access to the execution state at any block height
 which enables state retrieval at any point in history and beyond the execution node's pruning limit.
 
 The DPS also supports a set of custom smart contract resources that serve as wrapper for locked token vaults and as
 proxy to staking and delegating resources. This allows the DPS to track multiple balances per account, including locked,
 staked and delegated tokens, for accounts which deploy these custom resources.
 
+## Documentation
+
+### Binaries
+
+Below are links to the individual documentation for the binaries within this repository.
+
+* [`flow-dps-client`](./cmd/flow-dps-client/README.md)
+* [`flow-dps-indexer`](./cmd/flow-dps-indexer/README.md)
+* [`flow-dps-live`](./cmd/flow-dps-live/README.md)
+* [`flow-dps-server`](./cmd/flow-dps-server/README.md)
+* [`flow-rosetta-server`](./cmd/flow-rosetta-server/README.md)
+
+### API
+
+* [DPS API](./docs/dps-api.md)
+* [Access API](./docs/access-api.md)
+* [Rosetta API](./docs/rosetta-api.md)
+
+### Developer Documentation
+
+* [Introduction](./docs/introduction.md)
+* [Architecture](./docs/architecture.md)
+* [Database Schema](./docs/index.md)
+* [Snapshots](./docs/snapshots.md)
+
 ## Dependencies
 
 Go `v1.16` or higher is required to compile `flow-dps`.
```

Comment:
No, it's still `v1.16` or higher.

Reply:
No, the project can still be built using `1.16`. `1.17` is compatible too but not required, whereas building with `1.15` would fail because we use `io` instead of the deprecated `ioutil` package.